msb/serial/SerialReader.py

```diff
@@ -0,0 +1,83 @@
+import re
+import serial
+from serial.serialutil import SerialException
+import sys
+
+from msb.serial.config import SerialConf
+from msb.config import load_config
+from msb.zmq_base import get_default_publisher
+
+
+class SerialReader:
```

Comment:
Consider reordering the methods to make it easier to follow. I.e. ascending or descending in call order.